src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewGuidCommand.cs

```diff
@@ -0,0 +1,26 @@
+
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The implementation of the "new-guid" cmdlet
+    /// </summary>    
+    [Cmdlet(VerbsCommon.New, "Guid", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526920")]
+    [OutputType(typeof(Guid))]
+    public class NewGuidCommand : Cmdlet
+    {      
+        /// <summary>
+        /// returns a guid
+        /// </summary>
+        protected override void EndProcessing()
```

Comment:
This is intentional, see this discussion: https://github.com/PowerShell/PowerShell/pull/2786/files/48a6fb2e05c6d4954f94d9f34139ff338aae41a2#r89893087